Corona-Warn-App/src/main/res/layout/new_release_info_item.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout>
+    <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+        <TextView
+            android:id="@+id/title"
+            style="@style/subtitleBoldSixteen"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/spacing_normal"
+            android:layout_marginTop="@dimen/spacing_small"
+            android:layout_marginEnd="@dimen/spacing_large"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="@id/bullet_point"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="Title bullet point" />
+
+        <ImageView
+            android:id="@+id/bullet_point"
+            android:layout_width="@dimen/bullet_point_size"
+            android:layout_height="@dimen/bullet_point_size"
+            android:layout_marginStart="@dimen/spacing_normal"
+            android:layout_marginTop="@dimen/spacing_tiny"
+            android:importantForAccessibility="no"
+            android:src="@drawable/bullet_point"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="@id/title" />
```

Comment:
Could it be that the previous layout had more bottom padding between the graphic and the title? Can you compare with mock ups?

Reply:
I'm not sure if I understand what you mean. In the previous version the bullet point was centered, but that does not work with two-line headings.

Reply:
I don't think either looks bad, but I'm not sure if the change was intentional.

![Screenshot from 2021-01-29 15-17-41](https://user-images.githubusercontent.com/1439229/106285932-5bf54c00-6245-11eb-8fac-3cbe81cc32f8.png)
![Screenshot from 2021-01-29 15-18-04](https://user-images.githubusercontent.com/1439229/106285935-5c8de280-6245-11eb-986d-ad3f534a6a16.png)

Reply:
yes, I updated the illustration, so both have the same dimensions.